qsirecon/workflows/recon/amico.py

```diff
@@ -150,7 +177,9 @@ def init_amico_noddi_fit_wf(
         derivatives_config = load_yaml(load_data("nonscalars/amico_noddi.yaml"))
         ds_fibgz = pe.Node(
             DerivativesDataSink(
-                dismiss_entities=("desc",), compress=True, **derivatives_config["fibgz"]["bids"]
+                dismiss_entities=["desc"],
+                compress=True,
+                **derivatives_config["fibgz"]["bids"]
```

Comment:
```suggestion
                **derivatives_config["fibgz"]["bids"],
```
To satisfy the linter.

Reply:
Ah... I couldn't figure out what it was mad about